guides/common/modules/proc_generating-host-monitoring-reports.adoc

```diff
@@ -7,13 +7,7 @@ To schedule reports, configure a cron job or use the {ProjectWebUI}.
 .Procedure
 . In the {ProjectWebUI}, navigate to *Monitor* > *Reports* > *Report Templates*.
 For example, the following templates are available:
-Host {endash} Installed Products:: Use this template for hosts in Simple Content Access (SCA) organizations.
-It generates a report with installed product information along with other metrics included in *Subscription {endash} Entitlement Report* except information about subscriptions.
-
-Subscription {endash} Entitlement Report:: Use this template for hosts that are not in SCA organizations.
-It generates a report with information about subscription entitlements including when they expire.
-It only outputs information for hosts in organizations that do not use SCA.
-
+Host {endash} Installed Products:: This generates a report with installed product information along with other metrics, including system purpose attributes.
 . To the right of the report template that you want to use, click *Generate*.
 . Optional: To schedule a report, to the right of the *Generate at* field, click the icon to select the date and time you want to generate the report at.
 . Optional: To send a report to an e-mail address, select the *Send report via e-mail* checkbox, and in the *Deliver to e-mail addresses* field, enter the required e-mail address.
```

Comment:
Later in this file (Github won't let me comment on the specific line, but it starts on line 39) - We just moved the 'Days from Now' input from the entitlements report to the 'Subscription - General' report. So "Entitlement" should be changed to "General."